analyzer/src/analysis/value/interprocedural/sequential/function_fixpoint.cpp

```diff
@@ -76,7 +77,17 @@ FunctionFixpoint::FunctionFixpoint(
       _checkers(checkers),
       _exit_invariant(make_bottom_abstract_value(ctx)),
       _return_stmt(nullptr),
-      _logger(logger) {}
+      _logger(logger)
+{
+  if (_ctx.opts.trace_ar_statements) {
+    ar::TextFormatter formatter{};
+    auto& stream = analyzer::log::msg().stream();
+    stream << "\n>>>>>>>>>>>>>>\nEntering Interprocedural Sequential FunctionFixpoint for ";
+    ar::Namer namer;
+    formatter.format_header(stream, _function, namer);
+    stream << std::endl;
```

Comment:
Hmm, should we avoid `std::endl` because it does a flush? I don't know if we care. The logger implementation seems to avoid it: https://github.com/NASA-SW-VnV/ikos/blob/master/analyzer/include/ikos/analyzer/util/log.hpp#L247-L285

Reply:
> Hmm, should we avoid `std::endl` because it does a flush? I don't know if we care.

My main use case for the trace is when stepping through IKOS in the debugger. In that case I want to see each trace output as it happens. So flushing is desired.